{{cookiecutter.github_project_name}}/js/package.json

```diff
@@ -20,21 +20,30 @@
     "dist/*.js"
   ],
   "scripts": {
-    "clean": "rimraf dist/",
-    "prepublish": "webpack",
-    "build": "webpack",
+    "clean": "rimraf dist/ && rimraf ../{{ cookiecutter.python_package_name }}/labextension/ && rimraf ../{{ cookiecutter.python_package_name }}/nbextension",
+    "prepublish": "npm run clean && npm run build",
+    "build": "webpack && npm run build:labextension",
+    "build:labextension": "jupyter labextension build .",
     "watch": "webpack --watch --mode=development",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
-    "webpack": "^3.5.5",
+    "@jupyterlab/builder": "^3.0.0-rc.4",
+    "webpack": "^4.41.5",
```

Comment:
For the reader, if you are using webpack 3 in your widget package, it's important to update it to `4.*.*`

Reply:
should it be bumped to `^5` directly? or would the existing config have to be updated / migrated ?

Reply:
I picked 4 because ipyleaflet was using this version but we might be able to bump to 5, maybe the config still works.

Reply:
Note that it's important to update to Webpack 5, so we are in the same build conditions as in building the labextension, which will hopefully help to catch errors early.